src/configuration/validators/neovimValidator.ts

```diff
@@ -2,18 +2,37 @@ import { IConfiguration } from '../iconfiguration';
 import { IConfigurationValidator, ValidatorResults } from '../iconfigurationValidator';
 import { promisify } from 'util';
 import { execFile } from 'child_process';
+import * as path from 'path';
 
 export class NeovimValidator implements IConfigurationValidator {
   async validate(config: IConfiguration): Promise<ValidatorResults> {
     const result = new ValidatorResults();
 
     if (config.enableNeovim) {
+      let triedToParsePath = false;
       try {
+        // Try to find nvim in path if it is not defined
+        if (config.neovimPath === '') {
+          const pathVar = process.env.PATH;
+          if (pathVar) {
+            pathVar.split(';').forEach(element => {
```

Comment:
This is not going to work, on ubuntu the path would be /usr/bin/, I need to come up with another way